src/Form/CommitteeAroundAdherentType.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace AppBundle\Form;
+
+use AppBundle\Committee\CommitteeManager;
+use AppBundle\Entity\Committee;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\ChoiceType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormView;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class CommitteeAroundAdherentType extends AbstractType
+{
+    /** @var CommitteeManager $manager */
```

Comment:
This docbloc is not very usefull because this attribute is already declare in your construct.